internal/service/organization/resource_organization.go

```diff
@@ -82,6 +82,11 @@ func Resource() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+			"skip_default_alerts_settings": {
```

Comment:
is there a way to define the default value here? 

I could only find the following in the docs so just wondering if you know:
- https://developer.hashicorp.com/terraform/plugin/framework/migrating/attributes-blocks/default-values
- https://developer.hashicorp.com/terraform/plugin/framework/resources/default

Reply:
https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed 
The above clarifies that default cannot be used with Computed. 

The problem is that if backend ever returns a `false` for an org that may be created with <1.25 (assuming this would be released in 1.25), then:
- user config will not have this attr specified
- so TF will try to use the default `true` value
- during Read() TF will get `false` from the backend
- since we cannot mark the attribute as `Computed` if it also has `Default`, TF will plan to change the value to default `true` resulting in below non-empty plan for orgs created with older versions of our provider:
```

  ~ update in-place

Terraform will perform the following actions:

  # mongodbatlas_organization.test will be updated in-place
  ~ resource "mongodbatlas_organization" "test" {
        id                           = "b3JnX2lk:Njc3Mjk1Y2Y4ZTMyNWI0MjExOTVjMGJl"
        name                         = "manual-org-renamed-3-def"
      ~ skip_default_alerts_settings = false -> true
        # (10 unchanged attributes hidden)
    }
```

We may be able to define `Default: true` here ONLY if it can be guaranteed from the backend for all IaC orgs created with older versions of our provider to return `true` for this attribute. But we should not go this route because APIx 1 may do a one-time 'backfill' of this attr to `true` for IaC orgs but it may be possible in the future for users to use lower provider version for some reason and create new orgs. Does that make sense?